Application Actions/Sublime Text/1.0/Indent.scpt

```diff
@@ -0,0 +1,3 @@
+tell application "System Events" to tell process "Sublime Text"
+	key code 30 using command down
```

Comment:
This key code is also keyboard layout dependent – Any chance we can trigger a menu item click instead?